services/src/handlers/workflows.rs

```diff
@@ -63,6 +66,10 @@ where
     .service(
         web::resource("datasetFromWorkflow/{id}")
             .route(web::post().to(dataset_from_workflow_handler::<C>)),
+    )
+    .service(
+        web::resource("datasetFromWorkflowTask/{id}")
+            .route(web::post().to(dataset_from_workflow_handler_task::<C>)),
```

Comment:
Is it useful to have both methods?

Actually, we should only use a task for that, right?

Reply:
I removed it. f0fbc323499f3052b7b1663076599f7df685e88e